src/Http/ServerRequest.php

```diff
@@ -424,6 +426,9 @@ protected function _processGet($query, $queryString = '')
             $query += $queryArgs;
         }
 
+        parse_str($this->uri->getQuery(), $queryArgs);
```

Comment:
I think we should be able to remove the block that operates on `$queryString` here as the already has the query set, so we'd be parsing it twice now.